contribs/gnodev/cmd/gnodev/main.go

```diff
@@ -61,28 +59,39 @@ type devCfg struct {
 	webHTML             bool
 	webListenerAddr     string
 	webRemoteHelperAddr string
+	webWithHTML         bool
+	webHome             string
+
+	// Resolver
+	resolvers varResolver
 
 	// Node Configuration
-	minimal     bool
+	logFormat   string
+	lazyLoader  bool
 	verbose     bool
 	noWatch     bool
 	noReplay    bool
 	maxGas      int64
 	chainId     string
 	chainDomain string
-	serverMode  bool
 	unsafeAPI   bool
+	interactive bool
+	paths       string
 }
 
-var defaultDevOptions = &devCfg{
+var defaultDevOptions = devCfg{
 	chainId:             "dev",
-	chainDomain:         "gno.land",
+	logFormat:           "console",
+	chainDomain:         DefaultDomain,
 	maxGas:              10_000_000_000,
 	webListenerAddr:     "127.0.0.1:8888",
 	nodeRPCListenerAddr: "127.0.0.1:26657",
 	deployKey:           DefaultDeployerAddress.String(),
 	home:                gnoenv.HomeDir(),
 	root:                gnoenv.RootDir(),
+	interactive:         true,
```

Comment:
maybe default here should be based on [isatty](https://github.com/mattn/go-isatty).

Reply:
Good point! I already have the `termenv` library that should do the trick.

Reply:
good idea, done in d748ecf0df